pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MergeRollupMinionOfflineClusterIntegrationTest.java

```diff
@@ -65,9 +65,9 @@
 
 
 /**
- * Integration test for minion task of type "MergeRollupTask"
+ * Integration test for minion task of type "MergeRollupTask" configured on offline tables
  */
-public class MergeRollupMinionClusterIntegrationTest extends BaseClusterIntegrationTest {
+public class MergeRollupMinionOfflineClusterIntegrationTest extends BaseClusterIntegrationTest {
```

Comment:
By the way, is it possible to merge offline/realtime tests using 1 integration test since we are based on `BaseClusterIntegrationTest` for both tests? Integration test is expensive to set up the env.